include/godot_cpp/core/class_db.hpp

```diff
@@ -116,9 +116,13 @@ class ClassDB {
 	static void _register_class(bool p_virtual = false, bool p_exposed = true, bool p_runtime = false);
 
 	template <typename T>
-	static GDExtensionObjectPtr _create_instance_func(void *data) {
+	static GDExtensionObjectPtr _create_instance_func(void *data, GDExtensionBool p_notify_postinitialize) {
 		if constexpr (!std::is_abstract_v<T>) {
-			T *new_object = memnew(T);
+			Wrapped::_set_construct_info<T>();
```

Comment:
By replacing the `memnew` call with this, does the change imply that any Object created with `memnew` will not have `postinitialize` called? Should Godot-CPP users be using `ClassDB::instantiate` (or just this function) instead henceforth?

Reply:
This PR shouldn't change anything about how developers use godot-cpp, this should be a purely internal change within godot-cpp.

If you look a couple lines down, it will call `_postinitialize()`, but only if `p_notify_postinitialize` is `true`. We're not using `memnew()` because that should always lead to `NOTIFICATION_POSTINITIALIZE`, but here we need to be able to control whether we're doing it or not.

Reply:
Ahh that makes sense. Thanks for explaining!